Cargo.toml

```diff
@@ -7,6 +7,65 @@ members = [
 resolver = "2"
 exclude = ["axoproject/tests/projects/"]
 
+[workspace.package]
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/axodotdev/cargo-dist"
+homepage = "https://opensource.axo.dev/cargo-dist/"
+version = "0.17.0"
+
+[workspace.dependencies]
+# intra-workspace deps (you need to bump these versions when you cut releases too!
+cargo-dist-schema = { version = "=0.17.0", path = "cargo-dist-schema" }
+axoproject = { version = "=0.17.0", path = "axoproject", default-features = false, features = ["cargo-projects", "generic-projects"] }
```

Comment:
These 3 lines contain all copies of the current cargo-dist workspace that ever need to be edited.